collector/collector.go

```diff
@@ -82,7 +70,7 @@ func openDatabricksDatabase(config *Config) (*sql.DB, error) {
 // It orchestrates multiple specialized collectors for different metric categories.
 type Collector struct {
 	config       *Config
-	logger       log.Logger
+	logger       *slog.Logger
```

Comment:
Same, `promslog` configures a new `*slog.Logger` under the hood and returns it, this is the same logger.